tests/data/diagnosis_summary.json

```diff
@@ -463,7 +463,7 @@
         "vgg_models/pytorch-vgg19/steptime_train_float16": 45.2033062465,
         "vgg_models/pytorch-vgg19/throughput_train_float16": 709.1127328377,
         "diagnosis/accept": false,
-        "diagnosis/#issues": 1.0,
+        "diagnosis/issue_num": 1,
```

Comment:
@abuccts is it compitable?

Reply:
Why is related to this PR?

Reply:
> is it compatible?

the field is only used for human to read (in excel etc.)? not read by program

> Why is related to this PR?

just saw it's not consistent with others, could also in separate PR

Reply:
deleted renaming related logic to restore this PR, after thie pr being merged, I will submit another pr for that @abuccts